src/monitor.cpp

```diff
@@ -52,6 +53,7 @@ void updateGlobalData(void){
             
         //Make first API call to get global hash and current difficulty
         HTTPClient http;
+        try {
```

Comment:
Are you sure that this is correct way to handle errors here? I dont see that HTTPClient implementation raised any errors, nor DynamicJsonDocument. Deserialization errors usually handled with error code instead.

```
  DeserializationError error = deserializeJson(doc, resultString);
  if (error) {
```

Reply:
During my tests, I found out that the ESP-WROOM-32 has no memory to process the https request and handle the display like the ESP32-S3, so the .GET function crashes at diffent points. As a result, a piece of memory get lost each time, until it freezes or restart the esp in a few minutes.
So this "try" have the intention to free the alocated memory if a error has occurred.
In time, I changed the TFT_eSprite alocation , creating and detroing it before any http request, so the Httpclient find enough memory to work.
But if there´s a better way to do this, I´ll be happy to learn.